disnake/guild.py

```diff
@@ -3404,3 +3403,54 @@ async def bulk_edit_command_permissions(
             A list of edited permissions of application commands.
         """
         return await self._state.bulk_edit_command_permissions(self.id, permissions)
+
+    async def timeout(
+        self,
+        user: Snowflake = MISSING,
+        *,
+        seconds: Optional[int] = MISSING,
```

Comment:
```suggestion
        seconds: Optional[int],
```

`seconds` should be a required arg (with this, the handling of `MISSING` below isn't needed anymore)